lib/iiif_print/data/work_derivatives.rb

```diff
@@ -47,11 +47,11 @@ def self.data(from:, of_type:)
 
       # alternate constructor spelling:
       def self.of(work, fileset = nil, parent = nil)
-        new(work, fileset, parent)
+        new(work: work, fileset: fileset, parent: parent)
```

Comment:
This change in method signature breaks many of the other uses of the `IiifPrint::Data::WorkDerivatives.of` calls.  Please review the code base for those uses.

Reply:
Not trying to be combative, but how would it break them? For [example](https://github.com/scientist-softserv/iiif_print/blob/d9d139705a0e7923c5e544e7c99e417bad44a395/lib/iiif_print/data/work_file.rb#L77) this called #of like: 

```ruby
      def derivatives
        IiifPrint::Data::WorkDerivatives.of(work, fileset, self)
      end
```

the line we've changed should still work by assigning the parameters to the keyword assignments. Am I missing something? @jeremyf

Reply:
Hmm, there does seem to be a lot of broken specs in this area which seems to be pointing to method signature problems

Reply:
> Not trying to be combative, but how would it break them?

In my haste I mis-read the method signature.  You are correct.